main.pm

```diff
@@ -589,6 +589,10 @@ sub load_extra_test () {
 sub load_x11tests() {
     return unless (!get_var("INSTALLONLY") && get_var("DESKTOP") !~ /textmode|minimalx/ && !get_var("DUALBOOT") && !get_var("RESCUECD"));
 
+    if (!get_var("NOAUTOLOGIN")) {
```

Comment:
I don't see this as a generic X11 test, so I don't want to see this running on every staging prj. running this on an installed asset should do